sqlglot/dialects/doris.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+import typing as t
+
+from sqlglot import exp, generator
+from sqlglot.dialects.dialect import (
+    approx_count_distinct_sql,
+    arrow_json_extract_sql,
+    rename_func,
+)
+from sqlglot.dialects.mysql import MySQL
+from sqlglot.helper import seq_get
+
+
+def _to_date_sql(self: generator.Generator, expression: exp.TsOrDsToDate) -> str:
+    this = self.sql(expression, "this")
+    self.format_time(expression)
+    return f"TO_DATE({this})"
```

Comment:
Because the to_date() function in doris is the syntax https://doris.apache.org/zh-CN/docs/dev/sql-manual/sql-functions/date-time-functions/to_date, without parameters.

Reply:
this is not a valid link

Reply:
https://doris.apache.org/zh-CN/docs/1.2/sql-manual/sql-functions/date-time-functions/to_date/